providers/io/osf/normalizer.py

```diff
@@ -66,7 +66,7 @@ class Project(Parser):
         ctx.embeds.affiliated_institutions.data
     )
     date_updated = tools.ParseDate(ctx.attributes.date_modified)
-    subject = tools.Delegate(Tag, ctx.attributes.category)
+    subject = tools.Delegate(Tag, tools.Maybe(ctx, 'attributes.category'))
```

Comment:
Delete subject. It's being replaced soon and isn't currently used 🎉 
Maybe move it to tags? Map will filter out falsey values.